packages/codelabs/src/basics/web-components.md

```diff
@@ -45,31 +45,31 @@ To get started, let's create a basic HTML page:
 </html>
 ```
 
-If you run this in the browser and see hello world, you're good to go!
+If you run this in the browser and see hello world, then you're good to go!
 
 ## Custom Elements
 
-First, we will take a look at the most important web component API: custom elements.
+First, we will take a look at the most important web component API: `Custom Elements`.
```

Comment:
```suggestion
First, we will take a look at the most important web component API: Custom Elements.
```

The name of the specification is 'Custom Elements', but it's not code, so don't use backtics